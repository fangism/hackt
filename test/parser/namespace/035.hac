// "035.hac"
// copy-modified from "032.hac"
// child namespace cannot see un-exported definition in parent namespace
// expect fail

namespace ick {

defproc foo() { }

namespace earth {
	foo bar;
}	// end namespace earth
}	// end namespace ick
