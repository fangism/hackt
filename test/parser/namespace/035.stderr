Error: definition `ick::foo' is not exported from its home namespace.
concrete_type_ref: bad definition reference "foo"!  ERROR! [11:2..4]
